router/transformer/transformer_proxy_adapter.go

```diff
@@ -25,7 +25,8 @@ type transformerProxyAdapter interface {
 
 type ProxyRequestPayloadV0 struct {
 	integrations.PostParametersT
-	Metadata ProxyRequestMetadata `json:"metadata"`
+	Metadata          ProxyRequestMetadata   `json:"metadata"`
+	DestinationConfig map[string]interface{} `json:"destinationConfig"`
```

Comment:
This doesn't seem like a breaking change. So, no big deal. But ideally, we should be using v1 and deprecate v0 soon.